website/pages/blog/announcing-fets-client.mdx

```diff
@@ -0,0 +1,230 @@
+---
+title: Consume OpenAPI in TypeScript Without Code Generation
+authors: [aleksandra, arda]
+tags: [OpenAPI, TypeScript]
+date: 2023-04-24
+description:
+  FETS Client allows you to create an SDK-like client that infers types from an OpenAPI
+  specification document.
+image: /blog-assets/announcing-fets-client/cover.png
+thumbnail: /blog-assets/announcing-fets-client/thumbnail.png
+---
+
+Over the last few years, TypeScript, have been widely used to provide a type-safe development
+experience and de facto became a standard. However, generating types from an API typically requires
+a build step using a command-line interface (CLI) tool. Otherwise, you can write these types
+manually, but that means more work and can easily get outdated.
+
+We, The Guild, are a group of developers mainly working on the GraphQL ecosystem, but one of the
+biggest pain points we see in our users’ codebases is consuming REST APIs in a GraphQL BFF
+(backend-for-frontend).
+
+Even when you provide a well-crafted schema for the consumer (frontend) in GraphQL, finding
+solutions to maintain the same type-safe experience behind the scenes when communicating with
+underlying REST-based services can be challenging. The individual teams developing these services
+often have varying levels of expertise in GraphQL and use different languages and frameworks, such
+as Java, Spring Boot, or .NET.
+
+One common standard embraced by these teams is OpenAPI. Formerly known as Swagger, it become a
+standard within the REST community for describing APIs. It provides a comprehensive description of
+an API's capabilities, allowing consumers to understand its functionality without needing to know
+the details of its implementation.
+
+## 🔥 Introducing FETS Client 🔥
+
+We’re thrilled to show you what we’ve been building recently — a FETS Client that allows you to
+create an SDK-like client that infers types from an OpenAPI specification document.
+
+FETS Client is not just another code generator that requires an additional build step. Instead, it's
+a tool designed to work exclusively during build time, specifically during TypeScript transpilation.
+This means that it doesn't add the entire schema to your final code bundle, as it doesn't perform
+any runtime operations. As a result, FETS Client offers a lightweight and efficient solution for
+maintaining type safety in your code without impacting the size of your compiled assets.
+
+In summary, FETS Client means:
+
+😌 Type-safety out of the box — no CLI, no build step
+
+🚀 No runtime overhead — it’s super lightweight and performant
+
+💪 Support across different JavaScript environments, including Node.js, Deno, BUN, Cloudflare
+Workers, and AWS Lambda (it utilizes the web standard Fetch API)
+
+🔎 IDE features like `Go To Definition` to explore the API from TypeScript
+
```

Comment:
should we link already here to the comparison table?

Reply:
Not sure. It could make some people skip most of the blog post, and not seeing fets code examples 😅